lib/hedwig_slack/connection.ex

```diff
@@ -59,6 +59,7 @@ defmodule Hedwig.Adapters.Slack.Connection do
     for user <- decoded["users"] do
       send(state.owner, {:user, user})
     end
+    send(state.owner, :connection_ready)
```

Comment:
We should do this here instead: https://github.com/hedwig-im/hedwig_slack/blob/master/lib/hedwig_slack/connection.ex#L117